client/galaxy/scripts/mvc/workflow/workflow.js

```diff
@@ -0,0 +1,201 @@
+/** Workflow view */
+define( [ 'mvc/form/form-view',
+          'mvc/ui/ui-misc',
+          'mvc/ui/ui-select',
+          'mvc/tool/tool-form-composite' ], function( Form, Ui, Select, ToolForm ) {
```

Comment:
It doesn't look like we're actually using any of these here, or am I missing it?